File: CS/MySolution.Module.Win/Controllers/ControlViewItemControllerWin.cs
using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Layout;
using MySolution.Module.BusinessObjects;
using System;
using System.Windows.Forms;

namespace MySolution.Module.Win.Controllers {
    public class ControlViewItemControllerWin : ObjectViewController<DetailView, Contact>
    {
        protected override void OnActivated() {
            base.OnActivated();
            ControlViewItem item = ((DetailView)View).FindItem("MyButton") as ControlViewItem;
            if (item != null) {
                item.ControlCreated += item_ControlCreated;
            }
        }
        void item_ControlCreated(object sender, EventArgs e) {
            (((ControlViewItem)sender).Control as Button).Text = "Click me!";
            (((ControlViewItem)sender).Control as Button).Click += ButtonClickHandlingWinController_Click;
        }
        void ButtonClickHandlingWinController_Click(object sender, EventArgs e) {
            MessageBox.Show("Action from custom View Item was executed!");
        }
    }
}

File: VB/MySolution.Module.Win/Controllers/ControlViewItemControllerWin.vb
Imports DevExpress.ExpressApp
Imports DevExpress.ExpressApp.Layout
Imports MySolution.Module.BusinessObjects
Imports System
Imports System.Windows.Forms

Namespace MySolution.Module.Win.Controllers
    Public Class ControlViewItemControllerWin
        Inherits ObjectViewController(Of DetailView, Contact)

        Protected Overrides Sub OnActivated()
            MyBase.OnActivated()
            Dim item As ControlViewItem = TryCast(CType(View, DetailView).FindItem("MyButton"), ControlViewItem)
            If item IsNot Nothing Then
                AddHandler item.ControlCreated, AddressOf item_ControlCreated
            End If
        End Sub
        Private Sub item_ControlCreated(ByVal sender As Object, ByVal e As EventArgs)
            TryCast(DirectCast(sender, ControlViewItem).Control, Button).Text = "Click me!"
            AddHandler TryCast(DirectCast(sender, ControlViewItem).Control, Button).Click, AddressOf ButtonClickHandlingWinController_Click
        End Sub
        Private Sub ButtonClickHandlingWinController_Click(ByVal sender As Object, ByVal e As EventArgs)
            MessageBox.Show("Action from custom View Item was executed!")
        End Sub
    End Class
End Namespace